ts/features/fims/store/selectors/index.ts

```diff
@@ -0,0 +1,13 @@
+import { createSelector } from "reselect";
+import { GlobalState } from "../../../../store/reducers/types";
+
+const fimsFeatureSelector = (state: GlobalState) => state.features.fims;
+
+export const fimsConsentsSelector = createSelector(
```

Comment:
`createSelector` is not needed here, we are not performing expensive computations nor avoiding re-renderings